### PR TITLE
readding sidebar tips for everyone, added 3 more features to teams empty screen

### LINKED ISSUE #6474: [CAL-783] reshow sidebar tips
i made a whoopsy and by accident disabled the sidebar tips

<sub>[CAL-783](https://linear.app/calcom/issue/CAL-783/reshow-sidebar-tips)</sub>

### PR DESCRIPTION
**Pull request by PeerRich**
i did a whoopsy and accidentially hid the sidebar tips for app.cal.com (toHide was checking for a license -- which we obviously have)

also new while working on it:
CleanShot 2023-01-13 at 23 24 34@2x

### CODE REVIEW

File: packages/features/shell/Shell.tsx [hunk 1]
```diff
@@ -747,9 +747,7 @@ function SideBar() {
         </div>
 
         <div>
-          <LicenseRequired toHide>
-            <Tips />
-          </LicenseRequired>
+          <Tips />
```
Comment (zomars) [inline packages/features/shell/Shell.tsx:750]: Do we want these tips for self hosted users?

**Comment by emrysal** [comment]
NIT

image

File: packages/features/shell/Shell.tsx (hunk 1 above)
Comment (PeerRich) [reply packages/features/shell/Shell.tsx:750]: i think so, yea

Comment (PeerRich) [reply packages/features/shell/Shell.tsx:750]: they can remove it if needed

**Comment by PeerRich** [comment]
> NIT
> 
> image

urgh why

**Comment by PeerRich** [comment]
fixed 
CleanShot 2023-01-15 at 12 54 13@2x

**Review by emrysal — APPROVED** [review]
LGTM, looking good!

**Comment by emrysal** [comment]
I addressed some issues related to the CTA and header - CTA button height inconsistent throughout pages, layout errors comparing to the design, found out the NIT issue wasn't with your PR but instead was caused by the button being placed incorrectly everywhere

**Pull request merged**